qiskit/visualization/__init__.py

```diff
@@ -17,20 +17,160 @@
 
 .. currentmodule:: qiskit.visualization
 
+
+Overview
+========
+
+The visualization module contain functions that visualizes measurement outcome counts, quantum
+states, circuits, pulses, devices and more.
+
+Prerequisites
+=============
+
+To use visualization functions, you are required to install visualization optionals to your
+development environment:
+
+.. code-block:: bash
+
+   pip install 'qiskit[visualization]'
+
 Counts and State Visualizations
 ===============================
 
```

Comment:
I'd probably split this into "Counts Visualizations" and "State Visualizations", and get rid of all the nested headings within for the same reasons as the above points.  The APIs are already visually separated because they appear as tables.

Reply:
Make sense. Thanks for the suggestions. I was thinking do that but wasn't sure about splitting it because counts visualization contains only one function. But I agree with you now since we have api table and examples, it's better to split it to avoid extra headings.